src/main/java/org/folio/dataexp/service/export/ExportExecutor.java

```diff
@@ -75,11 +73,12 @@ private void updateJobExecutionStatusAndProgress(UUID jobExecutionId, ExportStra
     long exportsCompletedWithErrors = exports.stream().filter(e -> e.getStatus() == JobExecutionExportFilesStatus.COMPLETED_WITH_ERRORS).count();
     var currentDate = new Date();
     if (exportsCompleted + exportsFailed + exportsCompletedWithErrors == exports.size()) {
+      progress.setExported(commonExportStatistic.getExportedMarcListener().getExportedCount().get());
       if (Boolean.TRUE.equals(exportRequest.getAll())) {
         progress.setTotal(progress.getExported() - progress.getDuplicatedSrs() + progress.getFailed());
       }
-      progress.setFailed(progress.getFailed() + commonExportFails.getDuplicatedUUIDAmount() + commonExportFails.getInvalidUUIDFormat().size());
-      errorLogService.saveCommonExportFailsErrors(commonExportFails, progress.getFailed(), jobExecutionId);
+      progress.setFailed(progress.getFailed() + commonExportStatistic.getDuplicatedUUIDAmount() + commonExportStatistic.getInvalidUUIDFormat().size());
```

Comment:
Failed calculation can be extracted to separate method getFailedNumber.

Reply:
Updated thanks.